cratedb_jdbc/manifest.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='cratedb_jdbc' superclass='jdbc' plugin-version='0.0.1' name='CreateDB' version='18.1' min-version-tableau='2020.4'>
+  <vendor-information>
+      <company name="Crate.io"/>

```

Comment:
This results in a display:
![image](https://github.com/crate/tableau-connector/assets/37985796/831b8a74-448f-4b07-8163-ed26309a8ab4)

Are you alright with this @ckurze